Yoast/wordpress-seo
# Output optimized breadcrumbs

## Summary

This follows up on #13699 to output the optimized structure in both schema and the HTML breadcrumbs.

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Introduces the concept of memoizers to ensure presentations and the meta-tags context are not created multiple times during a single request.

## Test instructions

This PR can be tested by following these steps:

* See #13699 for how to setup a variety of posts with different breadcrumbs.
* Install a theme that uses our breadcrumbs.
* The breadcrumbs output should remain the same.

## UI changes

## Documentation

## Quality assurance
